_versions/2023-06-20-opensearch-2.8.0.markdown

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
This file already exists on `main`

Reply:
Thanks, wasn't sure if the protocol was to create a new file with a new date or modify the old file with the old date.
Changed in https://github.com/opensearch-project/project-website/pull/1730/commits/7cc73864ea73fb7aa9662e395dc50afa101a2751